packages/snfoundry/scripts-ts/deploy.ts

```diff
@@ -14,4 +14,18 @@ deployScript()
     exportDeployments();
     console.log("All Setup Done");
   })
-  .catch(console.error);
+  .catch((error) => {
```

Comment:
we should keep this section clean , the errors ( with their corresponding messages ) should be thrown on `deploy-contract.ts`